SINGLE.md

```diff
@@ -0,0 +1,161 @@
+# Single Execution
+
+## Approving the transaction
+
+### 1. Update repo and move to the appropriate folder for this rehearsal task:
+
+```
+cd superchain-ops
+git pull
+just install
+```
+
+### 2. Setup Ledger
+
+Your Ledger needs to be connected and unlocked. The Ethereum
+application needs to be opened on Ledger with the message "Application
+is ready".
+
+### 3. Simulate and validate the transaction
+
+Make sure your ledger is still unlocked and run the following command.
+
+**Note:** Remember that by default the script will assume the derivation path of your address is `m/44'/60'/0'/0/0`.
+If you wish to use a different account, append an `X` to the command to set the derivation path of the address that you want to use. For example by adding a `1` to the end, it will derive the address using `m/44'/60'/1'/0/0` instead.
+
+```shell
+just \
+   --dotenv-path tasks/<NETWORK_DIR>/<RUNBOOK_DIR>/.env \
+   --justfile single \
+   simulate \
+   tasks/<NETWORK_DIR>/<RUNBOOK_DIR> # 0 or 1 or ...
```

Comment:
One possible simplification is infer the dotenv-path from the task directory, so the command becomes:
```suggestion
just \
   --justfile single \
   simulate \
   tasks/<NETWORK_DIR>/<RUNBOOK_DIR> # 0 or 1 or ...
```

Reply:
Do you know how to do that with `just`? I haven't found a way yet, my current plan is to looking into it more, and to open an issue to ask if need be.

Reply:
figured out a decent approach.